plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/AbstractTrinoCatalog.java

```diff
@@ -120,6 +120,10 @@ public abstract class AbstractTrinoCatalog
         implements TrinoCatalog
 {
     public static final String TRINO_CREATED_BY_VALUE = "Trino Iceberg connector";
+    public static final String ICEBERG_VIEW_SQL_DIALECT = "trino";
+    public static final String ICEBERG_VIEW_RUN_AS_INVOKER = "trino_run_as_invoker";
+    public static final String ICEBERG_VIEW_OWNER = "trino_owner";
```

Comment:
We should document these somewhere so the spec isn't "in the code".  

Also, @electrum what do you think about inverting "run_as_invoker" to something like "run_as_owner".  That way when it is missing, we assume this is "invoker" which is what people normally want?  It should matter to anyone except those hacking the backing store.

Reply:
That makes sense to me. We also only need the single field `trino_run_as_owner` that contains the owner. If this is present, then it's a definer view with that owner. If empty, it's an invoker view. 

`ConnectorViewDefinition` has both a boolean and an optional owner for legacy reasons. In the engine, `ViewDefinition` has a single `Optional<Identity> runAsIdentity` field.